src/DIRAC/MonitoringSystem/private/DBUtils.py

```diff
@@ -156,3 +156,18 @@ def _getAccumulationMaxValue(self, dataDict):
                 if timeEpoch == maxEpoch:
                     maxValue += currentDict[timeEpoch]
         return maxValue
+
+    def _accumulate(self, granularity, startEpoch, endEpoch, dataDict):
```

Comment:
This can be a `staticmethod`

Reply:
BTW, which module calls this method?

Reply:
This is used by the plotters, specifically by the dataop. I mirrored what is done for Accounting